ocfweb/docs/docs/docs/constitution.md

```diff
@@ -240,14 +242,29 @@ constitution into compliance with requirements set by the University.
 ##### 8.1.2 Annotation
 Whenever this Constitution is amended, an annotation of the date of ratification shall be added to the beginning of this document and shall be further noted immediately following the new amendment.
 
-##### 8.1.3 ASUC Records    {amended-2015-09-01-part2}
-All amendments, additions or deletions to this document must be filed with the LEAD Center.
+##### 8.1.3 LEAD Center records    {amended-2016-05-xx-part0}
+All amendments, additions or deletions to this document must be filed with the
+LEAD Center in 432 Eshleman Hall.
 
-*[As amended at the General Meeting of September 1, 2015, with effect from June 26, 2015.]*
+*[Originally amended at the General Meeting of September 1, 2015, with effect from June 26, 2015.]*
+
+*[This section was modified by amendment on May xx, 2016.]*
 
 #### 8.2 Bylaws
 Bylaws may be created or modified as needed, through the [OCF Decision Making Process](#ocf_decision_making_process).
 
 
-### 9  Dissolution
-The assets of the OCF constitute a continuing trust for the benefit of all members of the Berkeley campus community interested in open computing facilities.  In the event of the dissolution of the OCF for whatever reason, the assets, after payment or adequate provision for payment of all outstanding debts and obligations of the OCF shall be transferred to a non-profit fund, foundation or corporation which is organized and operated exclusively for the purposes for which the OCF was founded.  All unspent ASUC funds shall remain the property of the ASUC.
+### 9  Dissolution    {amended-2016-05-xx-part1}
+The assets of the OCF constitute a continuing trust for the benefit of all members of the Berkeley campus community interested in open computing facilities.
```

Comment:
most of our constitution is not wrapped. i'm probably going to go fix that after this to make it consistently line wrapped.

in this case i didn't do it on that line to keep the diff more readable